src/Text.php

```diff
@@ -117,7 +99,23 @@ public function translate($string, $jsSafe = array(), $interpretBackSlashes = tr
 			}
 		}
 
-		return $lang->translate($string, $jsSafe, $interpretBackSlashes);
+		return $text->translate($string, $jsSafe, $interpretBackSlashes);
+	}
+
+	/**
+	 * Translates a string into the current language.
+	 *
+	 * @param   string   $string                The string to translate.
+	 * @param   boolean  $jsSafe                True to escape the string for JavaScript output
+	 * @param   boolean  $interpretBackSlashes  To interpret backslashes (\\=\, \n=carriage return, \t=tabulation)
+	 *
+	 * @return  string  The translated string or the key if $script is true
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function translate($string, $jsSafe = false, $interpretBackSlashes = true)
+	{
+		return $this->getLanguage()->translate($string, $jsSafe, $interpretBackSlashes);
```

Comment:
As this is a totally new method, I'd say don't worry about it.  The `_` method deals with the previous support.